src/main/java/io/appium/java_client/IOSMobileCommand.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Sounds good. I just didnt want to add uiaElement/:elementId/scroll to MobileCommands since this seemed more like a IOS command. I have no problem moving this to MobileCommand, actually make this easier.